distribution/docs/src/main/jdocs/content/_quickstart/quickstart-ingesting-contents.adoc

```diff
@@ -0,0 +1,33 @@
+:title: Ingesting (Quick Start)
+:type: quickStart
+:status: published
+:summary: Ingest sample data.
+:order: 02
+
+=== Ingesting (Quick Start)
+
+Now that ${branding} has been configured, ingest some sample data to demonstrate search capabilities.
+
+This is one way to ingest into the catalog, for a complete list of the different methods, see <<_ingesting_data,Ingesting Data>>.
+
+==== Ingesting Sample Data
+
+. Download a sample valid https://codice.atlassian.net/wiki/download/attachments/1179756/geojson_valid.json?version=1&modificationDate=1368249436010&api=v2[GeoJson file here].
+. Navigate in the browser to the Search UI at ${secure_url}/search.
+. Click the *upload* icon in the upper right corner.
+. Select the file to upload.
+
+[NOTE]
+====
+XML metadata for text searching is not automatically generated from GeoJson fields.
+====
+
+Querying from the Search UI (${secure_url}/search) will return the record for the file ingested:
```

Comment:
That isn't entirely true. If a security profile is selected, then the user would need privilege that was at least as high as the one selected, if and only if the thing being uploaded is not marked. For some profiles, it is nearly guaranteed that the user will have the correct privileges.